packages/esm-ward-app/src/hooks/useLocations.ts

```diff
@@ -0,0 +1,64 @@
+import { type FetchResponse, fhirBaseUrl, openmrsFetch } from '@openmrs/esm-framework';
+import { useEffect, useMemo, useState } from 'react';
+import useSWRImmutable from 'swr/immutable';
+
+interface FhirLocation {
+  fullUrl: string;
+  resource: {
+    resourceType: 'Location';
+    id: string;
+    name: string;
+    description: string;
+  };
+}
+
+interface FhirResponse {
+  resourceType: 'Bundle';
+  id: '6a107c31-d760-4df0-bb70-89ad742225ca';
+  meta: {
+    lastUpdated: '2024-08-08T06:28:01.495+00:00';
+  };
+  type: 'searchset';
+  total: number;
+  link: Array<{
+    relation: 'self' | 'prev' | 'next';
+    url: string;
+  }>;
+  entry: Array<FhirLocation>;
+}
+
+function getUrl(url: string) {
```

Comment:
What does this function do?

Reply:
I am not using this function anymore, thanks!